### PR TITLE
Budgets elaboration

### PR DESCRIPTION
**Pull request by ferblape**
Connects to #962

**WIP**: for the moment this is just the markup that needs to be styled and implemented.

### Pending things

- [x] Adjust table headers position:

screen shot 2017-09-23 at 13 49 38

- [x] Maybe we need a spinner when clicking between classifications? 
- [x] Metric boxes
- [x] Replace Summary by Budgets
- [x] Remove Budgets old action
- [x] user budget lines feedback

### How to activate

```ruby
s = Site.first
s.module_settings.create module_name: 'GobiertoBudgets', settings: { budgets_elaboration: true }
```

Also, run bubbles script in production.

### What does this PR do?

This PR implements the budgets elaboration section.

### How should this be manually tested?

Visit `/presupuestos/elaboracion`, and check everything is working.

### CODE REVIEW

**Comment by ferblape** [comment]
This is ready to review. I have updated the PR with the three things left.

File: app/controllers/gobierto_budgets/budgets_controller.rb [hunk 1]
```diff
@@ -14,11 +14,6 @@ def index
     @place_budget_lines = GobiertoBudgets::BudgetLine.all(where: { site: current_site, place: @place, level: 1, year: @year, kind: @kind, area_name: @area_name })
     @interesting_expenses = GobiertoBudgets::BudgetLine.all(where: { site: current_site, place: @place, level: 2, year: @year, kind: GobiertoBudgets::BudgetLine::EXPENSE, area_name: @interesting_area })
 
-    @main_budget_lines_summary = GobiertoBudgets::SiteStats.new(site: current_site, year: @year).main_budget_lines_summary
-
-    @any_custom_income_budget_lines  = GobiertoBudgets::BudgetLine.any_data?(site: current_site, kind: GobiertoBudgets::BudgetLine::INCOME,  area: GobiertoBudgets::CustomArea, index: GobiertoBudgets::SearchEngineConfiguration::BudgetLine.index_forecast)
```
Comment (amiedes) [inline app/controllers/gobierto_budgets/budgets_controller.rb:19]: @ferblape Why remove this?

Comment (ferblape) [reply app/controllers/gobierto_budgets/budgets_controller.rb:19]: Those variables are not used anymore in the new home, belong to the old home.

**Comment by ferblape** [comment]
@furilo I've created the data boxes in elaboration section. Check here: http://santfeliu.gobify.net/presupuestos/elaboracion. I've also enabled the bubbles again.

**Comment by ferblape** [comment]
I'm removing the WIP tag, I think this can be released to production @furilo

**Pull request merged**